site/gatsby-config.js

```diff
@@ -0,0 +1,13 @@
+const { plugins } = require('./src/config/base-gatsby-plugins');
+
+module.exports = {
+  siteMetadata: {
```

Comment:
Fill our `title` and `description` for this project similar to the [ComposeUI](https://github.com/morganstanley/ComposeUI/blob/b3a21e454f785a8e4825f6f39e810876c1c002c2/site/gatsby-config.js) config.